pkg/client/messaging/client_test.go

```diff
@@ -606,8 +606,9 @@ func TestCommand_Reply(t *testing.T) {
 
 		go func() {
 			for {
-				if len(registrar.Services()) > 0 {
-					_, e := registrar.Services()[0].HandleInbound(
+				svcs := registrar.Services()
```

Comment:
Note: this is a fix for an intermittent unit test failure, where the `if` block was entered, but the services list was somehow empty within (I presume it's a concurrency issue).

Reply:
This doesn't really make sense to me.